ext/native/gfx_es2/draw_text.cpp

```diff
@@ -34,7 +34,7 @@ void TextDrawer::SetFontScale(float xscale, float yscale) {
 }
 
 float TextDrawer::CalculateDPIScale() {
-	float scale = g_dpi_scale;
+	float scale = g_dpi_scale_x;
```

Comment:
`CtrlDisplayListView.cpp` used y, I think we should pick one and use it for any text scaling.  I have no strong opinion on which.

-[Unknown]